### PR TITLE
Update README with new eval output

### PR DESCRIPTION
**Pull request by danmcp**
- Includes new eval output
- Removes reference to GGUF not being supported for mmlu.  Was fixed in 0.19
- Removed separate examples for GGUF since they show the same output and are run the same way
- Uses a better judge model that doesn't have a high error rate
- Shows full mmlu output example
- Remove MIN_TASKS examples from main readme.  It's a development flag and shouldn't be highlighted as part of the main experience.

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Functional tests have been added, if necessary.
- [ ] E2E Workflow tests have been added, if necessary.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -763,155 +763,124 @@ You can use the `ilab model evaluate` command to evaluate the models you are tra
...
-
-### AVERAGE:
-0.45 (across 3)
-
-### SCORES:
-mmlu_abstract_algebra - 0.35
-mmlu_anatomy - 0.44
-mmlu_astronomy - 0.55
+## MODEL (SCORE)
+/home/user/.cache/instructlab/models/instructlab/granite-7b-test/ (0.52/1.0)
```
Comment (alimaredia) [inline README.md:782]: I noticed you added trailing slash at the end of `granite-7b-test/` and before there were none. Is that how the output looks or was there some other reason?

Comment (danmcp) [reply README.md:782]: Good catch!  I had it on the input side when I ran it to it was shown on the output.  Fixed.

File: README.md [hunk 2]
```diff
@@ -763,155 +763,124 @@ You can use the `ilab model evaluate` command to evaluate the models you are tra
...
 ...
 # SKILL EVALUATION REPORT
 
-## MODEL
-/home/example/.local/share/instructlab/models/granite-7b-lab-Q4_K_M.gguf
-
-### AVERAGE:
-5.0 (across 1)
+## MODEL (SCORE)
+/home/user/.cache/instructlab/models/instructlab/granite-7b-test/ (7.27/10.0)
```
Comment (alimaredia) [inline README.md:875]: There's still a trailing slash here

File: README.md [hunk 3]
```diff
@@ -763,155 +763,124 @@ You can use the `ilab model evaluate` command to evaluate the models you are tra
...
 $ export ILAB_MODELS_DIR=$HOME/.local/share/instructlab/models
-$ ilab model evaluate --benchmark mmlu_branch --model $ILAB_MODELS_DIR/instructlab/granite-7b-lab --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab
+$ ilab model evaluate --benchmark mmlu_branch --model $ILAB_MODELS_DIR/instructlab/granite-7b-test --base-model $ILAB_MODELS_DIR/instructlab/granite-7b-lab
 ...
 # KNOWLEDGE EVALUATION REPORT
 
-## BASE MODEL
-/home/example-user/.local/share/instructlab/models/instructlab/granite-7b-lab
+## BASE MODEL (SCORE)
+/home/user/.cache/instructlab/models/instructlab/granite-7b-lab/ (0.74/1.0)
```
Comment (alimaredia) [inline README.md:855]: Another trailing slash here

File: README.md (hunk 1 above)
Comment (alimaredia) [reply README.md:782]: I don't see any change, maybe you forgot to push your changes?

Your change just makes `/home/user/.cache/instructlab/models/instructlab/granite-7b-test/ (0.52/1.0)` into `/home/user/.cache/instructlab/models/instructlab/granite-7b-test (0.52/1.0)` right?

Comment (danmcp) [reply README.md]: Sorry, I pushed the wrong branch, should be fixed now.

File: README.md (hunk 2 above)
Comment (danmcp) [reply README.md]: Fixed, thanks!

File: README.md (hunk 3 above)
Comment (danmcp) [reply README.md]: Fixed, thanks!

**Review by alimaredia — APPROVED** [review]

**Review by nathan-weinberg — APPROVED** [review]

**Pull request merged**